ch_tools/chadmin/cli/chadmin_group.py

```diff
@@ -0,0 +1,62 @@
+from functools import wraps
+from typing import Optional
+
+import click
+import cloup
+
+from ch_tools import __version__
+from ch_tools.common import logging
+
+# pylint: disable=too-many-ancestors
+
+
+class Chadmin(cloup.Group):
+    def add_command(
+        self,
+        cmd: click.Command,
+        name: Optional[str] = None,
+        section: Optional[cloup.Section] = None,
+        fallback_to_default_section: bool = True,
+    ) -> None:
+        if cmd.callback is None:
+            super().add_command(
+                cmd,
+                name=name,
+                section=section,
+                fallback_to_default_section=fallback_to_default_section,
+            )
+            return
+
+        cmd_callback = cmd.callback
+
+        @wraps(cmd_callback)
+        @cloup.pass_context
+        def wrapper(ctx, *a, **kw):
+            logging.configure(
+                ctx.obj["config"]["loguru"], "chadmin", {"cmd_name": cmd.name}
+            )
+
+            logging.debug(
+                "Executing command '{}', params: {}, args: {}, version: {}",
+                cmd.name,
+                {
+                    **ctx.parent.params,
+                    **ctx.params,
+                },
+                ctx.args,
+                __version__,
+            )
+
+            result = cmd_callback(*a, **kw)
```

Comment:
Do we log something in case of error/fail in `cmd_callback`?
Maybe add some common message like in `monrun_checks_keeper` [here](https://github.com/yandex/ch-tools/pull/210/files#diff-073ad2263dfe496edc132c2c57a8d2d146262e3e5da43bbd02d8c0a5b1c4e51aR73)?

Reply:
It was wrapped in `try...except`